src/content/translations/ro/history/index.md

```diff
@@ -140,16 +140,16 @@ Furculița Byzantium:
 - A redus recompensele [miniere](/developers/docs/consensus-mechanisms/pow/mining/) pe bloc de la 5 la 3 ETH.
 - A amânat [bomba de dificultate](/glossary/#difficulty-bomb) cu un an.
 - A adăugat capacitatea de a efectua apeluri către alte contracte care nu modifică starea.
-- A adăugat anumite metode de criptografie pentru a permite \[scalarea layer-ului 2\]((/developers/docs/layer-2-scaling/).
+- A adăugat anumite metode de criptografie pentru a permite \[scalarea layer-ului 2\]((/developers/docs/scaling/#layer-2-scaling).
 
 [Citește anunțul Fundației Ethereum](https://blog.ethereum.org/2017/10/12/byzantium-hf-announcement/)
 
 <ExpandableCard title="EIP-uri Byzantium" contentPreview="Official improvements included in this fork.">
 
 - [EIP-140](https://eips.ethereum.org/EIPS/eip-140) – _adaugă opcodul „REVERT”._
 - [EIP-658](https://eips.ethereum.org/EIPS/eip-658) – _adaugă câmpul „stare” la recepțiile tranzacției pentru a indica succesul sau eșecul_
-- [EIP-196](https://eips.ethereum.org/EIPS/eip-196) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/layer-2-scaling/#rollups)._
-- [EIP-197](https://eips.ethereum.org/EIPS/eip-197) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/layer-2-scaling/#rollups)._
+- [EIP-196](https://eips.ethereum.org/EIPS/eip-196) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/scaling/layer-2-rollups/#rollups-and-sidechains)._
+- [EIP-197](https://eips.ethereum.org/EIPS/eip-197) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/scaling/layer-2-rollups/#rollups-and-sidechains)._
```

Comment:
`#rollups-and-sidechains` does not exist as a header link. The existing internal link of `#rollups` should be used here.